spec/core/board_spec.rb

```diff
@@ -6,6 +6,12 @@
 
   let(:board) { Board.new(3) }
 
+  def mark_board(marks)
+    for position in marks
+      board.mark(position, "X")
+    end
+  end
+
```

Comment:
This helper method is duplicated between two files. Can you refactor your tests to pull it out into its own file?